pyscriptjs/src/plugins/pyterminal.ts

```diff
@@ -89,38 +130,117 @@ function make_PyTerminal(app: PyScriptApp) {
             this.outElem.className = 'py-terminal';
```

Comment:
wouldn't it be better to just use `classList.add('py-terminal')` so that we don't erase or override possible user-land classes added to theme somehow a terminal?